docs/gettingstarted/structuring.md

```diff
@@ -24,8 +24,8 @@ The `mods.toml` file is formatted as [TOML](https://github.com/toml-lang/toml),
     # The name of the mod loader type to load - for regular FML @Mod mods it should be javafml
     modLoader="javafml"
     # A version range to match for said mod loader - for regular FML @Mod it will be the forge version
-    # Forge for 1.15.2 is version 31
-    loaderVersion="[31,)"
+    # Forge for 1.16.3 is version 34*. It is recommended to leave a comma at the end to support future versions
```

Comment:
Why the random asterisk?
```suggestion
    # Forge for 1.16.3 is version 34. It is recommended to leave a comma at the end to support future versions
```

Reply:
It is to point the reader to the note below `* All version ranges use...`

Reply:
No idea how markdown/rtd deals with symbols but would a little superscript ¹ do the job there? Assuming there's no more superscripts in the page. Otherwise ²³⁴⁵⁶⁷⁸⁹⁰ are also available :P

Reply:
No, that's just confusing. This is a code example. If the note below is important it should be in an admonition, not using custom HTML.